simtools/db_handler.py

```diff
@@ -176,7 +178,7 @@ def exportModelFiles(self, parameters, dest):
 
         if cfg.get("useMongoDB"):
             self._logger.debug("Exporting model files from MongoDB")
-            for par, info in parameters.items():
+            for _, info in parameters.items():
```

Comment:
In the case you changed below, you changed it instead to iterate only on the values. It's worth being consistent I think and the other solution is nicer so I suggest to implement it here as well.

Reply:
Agree. 0f918a3